pkgs/build-support/go/gomod2nix/default.nix

```
@@ -0,0 +1,431 @@
+{ stdenv
+, stdenvNoCC
+, runCommand
+, buildEnv
+, lib
+, fetchgit
+, jq
+, cacert
+, pkgs
+, pkgsBuildBuild
+, runtimeShell
+, writeScript
+, gomod2nix
+}:
+let
+
+  inherit (builtins) substring toJSON hasAttr trace split readFile elemAt;
+  inherit (lib)
+    concatStringsSep replaceStrings removePrefix optionalString pathExists
+    optional concatMapStrings fetchers filterAttrs mapAttrs mapAttrsToList
+    warnIf optionalAttrs platforms
+    ;
+
+  parseGoMod = import ./parser.nix;
+
+  removeExpr = refs: ''remove-references-to ${concatMapStrings (ref: " -t ${ref}") refs}'';
```

```suggestion
  removeExpr = refs: "remove-references-to ${concatMapStrings (ref: " -t ${ref}") refs}";
```

the other normal quotes are within `${}` so it should be fine to just use normal quotes

---

This entire line is unused now anyway so I've removed it.